fighthealthinsurance/chat/tools/prior_auth_tool.py

```
@@ -0,0 +1,184 @@
+            )
+            await self.send_status_message(
+                "Error processing prior auth data: Invalid JSON format."
+            )
+            raise
+
+        except Exception as e:
+            logger.opt(exception=True).warning(
+                f"Error processing prior auth data: {e}"
+            )
+            await self.send_status_message(
+                f"Error processing prior auth data: {str(e)}"
```

Ruff (0.14.10)

Use explicit conversion flag

Replace with conversion flag

(RUF010)